apps/condo/index.js

```diff
@@ -62,7 +56,13 @@ const keystone = new Keystone({
     },
 })
 
-if (!IS_BUILD_PHASE) {
+if (IS_BUILD_PHASE) {
+    // NOTE: we need to register User list for `createAuthStrategy`
+    //  and we need the AuthStrategy because it's required for /admin/ static files build
+    registerSchemas(keystone, [
+        require('@core/keystone/schemas/User'),
+    ])
```

Comment:
Why during build phase other schemas are omitted?
Why only `User` schema is registered?
Why it is registered from core?

Reply:
Could you write extended documentation about that?

Reply:
Look at `NOTE` comment on line 60